circle.yml

```diff
@@ -5,8 +5,27 @@ test:
   pre:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
-    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make build-server ;; esac:
-        parallel: true
+    - ? |
+          case $CIRCLE_NODE_INDEX in
+            0) NODE_ENV=test make build-server;
+               ;;
+
+            1) I18N_DIR=$CIRCLE_ARTIFACTS/translate;
```

Comment:
I think this will never run if we've only got a single instance available.  It looks like `CIRCLE_NODE_TOTAL` is the environment variable to check for parallelism, so I think we need to put it after this case statement in a separate if, like:

`if [ "$CIRCLE_NODE_TOTAL" == 1 ] || [ "$CIRCLE_NODE_INDEX" == 1 ] ; then …; fi`

That way, when there's only one container available, they'll run sequentially. When there's more than one container, the behaviour is basically the same as the case statement. We run after the build-server step, but it's a no-op on ever other container, so that's fine.

I'm not sure whether we should check for empty too (`[ -z "$CIRCLE_NODE_TOTAL" ]`).